tianshou/highlevel/env.py

```diff
@@ -72,14 +71,18 @@ class VectorEnvType(Enum):
     RAY = "ray"
     """Parallelization based on the `ray` library"""
 
-    def create_venv(self, factories: Sequence[Callable[[], gym.Env]]) -> BaseVectorEnv:
+    def create_venv(
+        self,
+        factories: Sequence[Callable[[], gym.Env]],
+        venv_kwargs: dict,
```

Comment:
this is also breaking, default to None?

Reply:
Generally, I'm not a huge fan of passing kwargs - one could also solve this with VenvFactories. It would be somehow more in line with the rest of the code, although for this particular case it may also be an overkill.

@opcode81 what do you think?

Reply:
I made all the kwargs in venvs explicit with default values. At least it shouldn't be breaking anymore?

Reply:
Not a fan of passing kwargs either.
Which kwargs might a user reasonably pass here? Can you provide some examples? Maybe we can directly map all the reasonable options to a separate VectorEnvType enum item.

Reply:
Correct, it's mainly about `context`. It's only special cases such as the aforementioned parallelization under MacOS where you would explicitly like to set a context as the default `None` simply uses the system default. A `SUBPROC_SHARED_MEM_FORK_CONTEXT` could do the trick, not exposing theses options to the user

Reply:
kwargs are gone and backwards compat restored, resolving